### PR TITLE
add background removing from individual beetle image

### PR DESCRIPTION
**Pull request by kwadraterry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,5 +54,11 @@ python3 individual_beetles.py --images <path to group_images> --csv <path to met
 
 **FYI**: The script to crop out individual beetles works well for the images that have coords_pix_length and coords_pix_width information correctly align to beetles. However, there are a couple images where this is not the case, and thus the segmentation of beetles will not result in a nice crop of the individual beetles.
 
+To remove the background from the individual images, run:
```
Comment (hlapp) [inline README.md:57]: Does this segment the beetle images, ore remove the background, or perhaps both? Either way, either the documentation or the name of the script or both should be changed to be consistent.

Comment (egrace479) [reply README.md:57]: I believe it's just background removal from the segmented individual images. @kwadraterry, if you want to keep it under the segmentation folder, just rename the file to something like `remove_background.py` or `remove_individual_background.py`.

Comment (ramirezmichelle) [reply README.md:57]: we already have a remove_background.py, so something like remove_individual_background.py would be best

**Review by ramirezmichelle — APPROVED** [review]

**Review by egrace479 — APPROVED** [review]
LGTM

**Comment by hlapp** [comment]
@egrace479 @ramirezmichelle looks like this should be merged?

**Pull request merged**